dashboard/lib/client_state.rb

```diff
@@ -13,9 +13,8 @@ def initialize(session, cookies)
 
   # Resets all client state (level progress, lines of code, videos seen, etc.)
   def reset
-    # Also reset unencrypted cookies in case we are in a rolled-back state.
-    session[:lines] = cookies[:lines] = nil
-    session[:progress] = cookies[:progress] = nil
+    cookies[:lines] = nil
```

Comment:
These changes are now checked in; could you pull from staging so we don't see them?